src/components/ZoneSearchPanel.js

```diff
@@ -78,8 +80,26 @@ export function ZoneSearchPanel({ autoFocus, onSearchCompleted }) {
                         {z.province.name} เขตเลือกตั้งที่ {z.zone.no}
                       </b>
                     </div>
-                    <div>{z.zone.details}</div>
+                    <div
+                      css={{
+                        color: "gray",
+                      }}
+                    >
+                      {z.zone.details}
+                    </div>
                   </Link>
+                  <div
+                    css={{
+                      position: "absolute",
+                      right: "4px",
+                      top: "calc(50% - 3px)",
+                      border: "solid #212121",
+                      borderWidth: "0 2px 2px 0",
+                      // display: 'inline-block',
+                      padding: "4px",
+                      transform: "rotate(-45deg)",
```

Comment:
I really like how you managed to create a `>` icon without using any icon font!

Impressive~